dotnet/src/MicroFocus.FAS.AdapterSdkSchema.SchemaObjectBuilders/DateTimeFunctions.cs

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright 2022-2023 Open Text.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+using System;
+
+namespace MicroFocus.FAS.AdapterSdkSchema.SchemaObjectBuilders
+{
+    internal static class DateTimeFunctions
+    {
+        internal static string ToEpochSecondsString(DateTime value)
+        {
+            TimeSpan t = value - new DateTime(1970, 1, 1);
+            long secondsSinceEpoch = (long)t.TotalSeconds;
+            return secondsSinceEpoch.ToString();
```

Comment:
.NET has standard functionality for this.  I think it's `new DateTimeOffset(value).ToUnixTimeSeconds()`.